src/import/sst_service.rs

```diff
@@ -875,6 +875,11 @@ impl<E: Engine> ImportSst for ImportSstService<E> {
             sst_importer::metrics::IMPORTER_DOWNLOAD_DURATION
                 .with_label_values(&["queue"])
                 .observe(start.saturating_elapsed().as_secs_f64());
+            if get_disk_status(0) != DiskUsage::Normal {
```

Comment:
can we add this check in other interface? `fn apply()` also need this check.

Reply:
ok